### PR TITLE
feat: Use API to fill profile info

### LINKED ISSUE #141: Use API to fill information in profile page
Right now, the information in profile page is static.

What to do?
- Call `api/get-user` endpoint to get user's details
- Fill the username & image using the response of API
In case the image is null, show the default image only.
- The information for `joined-on` is not available right now. ( being worked on in https://github.com/project-travel-mate/server/issues/30 )

### PR DESCRIPTION
**Pull request by brijeshshah13**
# Description

Fetched the info from the `get-user` end-point to update the profile details.

screenshot_20180629-130958

Fixes #141 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by brijeshshah13** [comment]
@Swati4star I have parsed and set the joined date as well as it was implemented in the server today morning.

**Comment by Swati4star** [comment]
In the last line, the user might not realize that the given date is his joining date.
You can write something like... joined on 15Jun 18
There's no need to mention the time.

File: Android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -149,6 +149,7 @@
     <string name="text_book">Book</string>
     <string name="text_my_trips">My Trips</string>
     <string name="unknown_value">…</string>
+    <string name="text_joining_date">Joined on</string>
```
Comment (Swati4star) [inline Android/app/src/main/res/values/strings.xml]: Add space after `on`

Comment (brijeshshah13) [reply Android/app/src/main/res/values/strings.xml]: Done.

**Pull request merged**